src/main/java/seedu/fridgefriend/FridgeFriend.java

```diff
@@ -86,18 +93,40 @@ private static void printExceptionMessage(Exception exception) {
     }
 
     /**
-     * Parses description into food category and name.
+     * Define arguments format for add food command.
+     * A Pattern object which defines how the input string for food item
+     * that should look like. [^/]+ implies 1 or more characters except for '/'
+     */
+    public static final Pattern FOOD_DATA_ARGS_FORMAT =
+            Pattern.compile("(?<name>[^/]+)"
+                    + " /c (?<category>[^/]+)"
+                    + " /exp (?<expiryDate>[^/]+)"
+                    + " /loc (?<storageLocation>[^/]+)");
```

Comment:
Code looks good and clean. Well done!

Only request is to explain a little bit slightly more about what this Pattern Compile does, since it is a "rare" and technical usage. As long as it is explained once, no need to explain again in future, should be good to go.